core/capabilities/ccip/ccipsolana/msghasher.go

```diff
@@ -74,7 +90,7 @@ func (h *MessageHasherV1) Hash(_ context.Context, msg cciptypes.Message) (ccipty
 			DestTokenAddress:  solana.PublicKeyFromBytes(ta.DestTokenAddress),
 			ExtraData:         ta.ExtraData,
 			DestGasAmount:     destGasAmount,
-			Amount:            ccip_offramp.CrossChainAmount{LeBytes: tokens.ToLittleEndianU256(ta.Amount.Int.Uint64())},
+			Amount:            ccip_offramp.CrossChainAmount{LeBytes: toLittleEndian(ta.Amount.Int)},
```

Comment:
The function added last night was a duplicate and should reuse the existing function instead of making a new one:
```suggestion
			Amount:            ccip_offramp.CrossChainAmount{LeBytes: ([32]uint8)(encodeBigIntToFixedLengthLE(ta.Amount.Int, 32))},
```